docgen/src/guides/v3-migration.md

```diff
@@ -149,6 +149,84 @@ InstantSearch 3 introduces some breaking changes in the widget's naming, options
   <button class="ais-InfiniteHits-loadMore">Show more results</button>
 ```
 
+### RangeSlider
+
+#### Options
+
+| Before          | After       |
+| --------------- | ----------- |
+| `attributeName` | `attribute` |
+
+#### CSS classes
+
+| Before                                | After                                |
+| ------------------------------------- | ------------------------------------ |
+| `ais-range-slider`                    | `ais-RangeSlider`                    |
+| `ais-range-slider--disabled`          | `ais-RangeSlider--disabled`          |
+| `ais-range-slider--handle`            | `ais-RangeSlider-handle`             |
+| `ais-range-slider--tooltip`           | `ais-RangeSlider-tooltip`            |
+| `ais-range-slider--value`             | `ais-RangeSlider-value`              |
+| `ais-range-slider--marker-horizontal` | `ais-RangeSlider-marker--horizontal` |
+| `ais-range-slider--marker-large`      | `ais-RangeSlider-marker--large`      |
```

Comment:
Maybe we could document the rheostat classes that we are using in the stylesheet?

Reply:
The Rheostat classes haven't changed - this is a migration guide.

This is not the place for documenting them, right?

Reply:
Since we've never documented it, I thought it could have been a good thing but it's true this is not in the migration scope :) Let's keep it like that for the moment then.